Unable to resolve gradle-baseline-java 0.9.2

Did this artifact get removed from the bintray and gradle plugins repositories?

```
aash@aash01-mac ~/git/http-remoting$ ./gradlew idea

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'http-remoting'.
> Could not resolve all dependencies for configuration ':classpath'.
   > Could not find com.palantir:gradle-baseline-java:0.9.2.
     Searched in the following locations:
         https://jcenter.bintray.com/com/palantir/gradle-baseline-java/0.9.2/gradle-baseline-java-0.9.2.pom
         https://jcenter.bintray.com/com/palantir/gradle-baseline-java/0.9.2/gradle-baseline-java-0.9.2.jar
         https://plugins.gradle.org/m2/com/palantir/gradle-baseline-java/0.9.2/gradle-baseline-java-0.9.2.pom
         https://plugins.gradle.org/m2/com/palantir/gradle-baseline-java/0.9.2/gradle-baseline-java-0.9.2.jar
     Required by:
         project :

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 2.464 secs
aash@aash01-mac ~/git/http-remoting$ git status
On branch develop
Your branch is up-to-date with 'origin/develop'.
nothing to commit, working tree clean
aash@aash01-mac ~/git/http-remoting$
```

https://github.com/palantir/gradle-baseline/releases/tag/0.9.2

jcenter publishing is broken; try the direct bintray link (see readme)

On Thu, Apr 13, 2017 at 1:04 PM Andrew Ash <notifications@github.com> wrote:

> Did this artifact get removed from the bintray and gradle plugins
> repositories?
>
> aash@aash01-mac ~/git/http-remoting$ ./gradlew idea
>
> FAILURE: Build failed with an exception.
>
> * What went wrong:
> A problem occurred configuring root project 'http-remoting'.
> > Could not resolve all dependencies for configuration ':classpath'.
>    > Could not find com.palantir:gradle-baseline-java:0.9.2.
>      Searched in the following locations:
>          https://jcenter.bintray.com/com/palantir/gradle-baseline-java/0.9.2/gradle-baseline-java-0.9.2.pom
>          https://jcenter.bintray.com/com/palantir/gradle-baseline-java/0.9.2/gradle-baseline-java-0.9.2.jar
>          https://plugins.gradle.org/m2/com/palantir/gradle-baseline-java/0.9.2/gradle-baseline-java-0.9.2.pom
>          https://plugins.gradle.org/m2/com/palantir/gradle-baseline-java/0.9.2/gradle-baseline-java-0.9.2.jar
>      Required by:
>          project :
>
> * Try:
> Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
>
> BUILD FAILED
>
> Total time: 2.464 secs
> aash@aash01-mac ~/git/http-remoting$ git status
> On branch develop
> Your branch is up-to-date with 'origin/develop'.
> nothing to commit, working tree clean
> aash@aash01-mac ~/git/http-remoting$
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/palantir/http-remoting/issues/400>, or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGOdwZ4QGSkJGoxciHShAPttCXemcCgQks5rvn-8gaJpZM4M9JAl>
> .
>

--- a/build.gradle
+++ b/build.gradle
@@ -1,6 +1,7 @@
 buildscript {
     repositories {
         jcenter()
+        maven { url "http://palantir.bintray.com/releases" }
     }
 
     dependencies {

